.drone.yml

```diff
@@ -42,7 +53,7 @@ pipeline:
       TAGS: bindata
       GOPATH: /srv/app
     commands:
-      - make test-pgsql
+      - echo 'Not ready yet'
```

Comment:
Maybe you could leave the intended command in here, for reference, like `echo make 'test-pgsql #not ready yet'` (and same for all non-ready blocks)